<launch>
    <!-- Load joint controller configurations from YAML file to parameter server -->
    <rosparam file="$(find mico_hardware)/config/controllers.yaml" command="load"/>
    <node pkg="mico_hardware" 
          name="mico_hardware" 
          type="mico_hardware" 
          output="screen"/>
    <node pkg="controller_manager"
          type="spawner"
          args="pos_j1_controller 
          pos_j2_controller
          pos_j3_controller
          pos_j4_controller 
          pos_j5_controller 
          pos_j6_controller 
          pos_f1_controller 
          pos_f2_controller 
          joint_state_controller 
          position_joint_mode_controller"
          name="controller_manager"/>

</launch>
